pkg/minikube/cluster/commands.go

```diff
@@ -43,3 +43,5 @@ func GetStartCommand() string {
 	flags := strings.Join(flagVals, " ")
 	return fmt.Sprintf(startCommandFmtStr, flags, constants.RemoteLocalKubeErrPath, constants.RemoteLocalKubeOutPath)
 }
+
+var localkubeStatusCommand = fmt.Sprintf("ps aux | grep localkube | wc -l")
```

Comment:
I thought I commented, but I don't see it anymore. What do you think about switching this to a pidfile type system? During start, we could write a pidfile like this: https://github.com/fhd/init-script-template/blob/master/template#L41

Then to check the status it's easier, you can do: https://github.com/fhd/init-script-template/blob/master/template#L26